python/ray/tune/BUILD

```diff
@@ -304,7 +304,7 @@ py_test(
 
 py_test(
     name = "test_trial_scheduler_pbt",
-    size = "medium",
+    size = "large",
```

Comment:
Yes, as explained [here](https://github.com/ray-project/ray/pull/19515#discussion_r748323869), I have to change to `10, 20, 40` from `10, 20, 30` to avoid racing. As a result, the test time is elongated.

Reply:
Sorry, slipped my mind :-)